ui/admin/app/templates/scopes/scope/edit.hbs

```diff
@@ -66,21 +66,110 @@
+            </Hds::Text::Body>
+
+            {{#if @model.storage_policy_id}}
+              <LinkListPanel as |P|>
+                <P.Item
+                  @icon='hourglass'
+                  @color='boundary'
+                  @text={{this.storage_policy.name}}
+                >
+                  {{#if this.storage_policy.retain_for.days}}
+                    <Hds::Text::Body
+                      @tag='p'
+                      @color='var(--ui-gray-subtler-1)'
+                      @size='100'
+                    >
+                      {{t 'resources.policy.titles.retain_for'}}
+                      {{format-day-year this.storage_policy.retain_for.days}}
+                    </Hds::Text::Body>
+                  {{/if}}
+                  {{#if this.storage_policy.delete_after.days}}
+                    <Hds::Text::Body
+                      @tag='p'
+                      @color='var(--ui-gray-subtler-1)'
+                      @size='100'
+                    >
+                      {{t 'resources.policy.titles.delete_after'}}
+                      {{format-day-year this.storage_policy.delete_after.days}}
+                    </Hds::Text::Body>
+                  {{/if}}
+                </P.Item>
+                {{#if (can 'detachStoragePolicy scope' @model)}}
+                  <P.Item>
+                    <Hds::Dropdown as |dd|>
+                      <dd.ToggleIcon
+                        @icon='more-horizontal'
+                        @text={{t 'actions.manage'}}
+                        @hasChevron={{false}}
+                        @size='small'
+                      />
+                      <dd.Interactive
+                        @route='scopes.scope.policies.policy'
+                        @model={{this.storage_policy.id}}
+                        @text={{t 'actions.view'}}
+                      />
+                      <dd.Separator />
+                      <dd.Interactive
+                        @text={{t 'resources.policy.actions.delete'}}
+                        @color='critical'
+                        {{on
+                          'click'
+                          (route-action 'detachStoragePolicy' @model)
+                        }}
+                      />
+                    </Hds::Dropdown>
+                  </P.Item>
+                {{/if}}
+
+              </LinkListPanel>
+
+              <Hds::Link::Standalone
+                @icon='arrow-right'
+                @iconPosition='trailing'
+                @text={{t 'resources.policy.actions.add'}}
+                @route='scopes.scope.add-storage-policy'
+              />
+
+            {{else}}
+              <Hds::Button
+                @text={{t 'resources.policy.actions.add'}}
+                @color='secondary'
+                @route='scopes.scope.add-storage-policy'
+              />
+            {{/if}}
+          </div>
+        </bc.Sidebar>
+      {{/if}}
+    </Rose::Layout::BodyContent>
   </page.body>
```

Comment:
From my understanding, attaching a storage policy to the global scope is possible. So I'm wondering if a nav link needs to be added with label "Global settings" so user can navigate and add a storage policy (similar to Org Settings nav link).

Reply:
This will be part of a separate [ticket](https://hashicorp.atlassian.net/browse/ICU-12257)